### DISCUSSION TITLE
Migrate from `dart:html` to `package:web`🚂

### DISCUSSION THREAD

**Discussion by Dan-Crane** [cfug/dio#2163] [General]
Hi!:)

The flutter masters advise to switch from the `dart:html` to the `package:web`. How soon is this going to happen?

UPD:
Here is what the masters say: [[TAP]](https://dart.dev/interop/js-interop/package-web#:~:text=If%20you%20maintain,for%20more%20information.)

**Comment by AlexV525** [comment]
`web` requires Dart 3.3. So unless we abstract the web adapter to a standalone package, we'd probably not to migrate until `dart:html` is removed or we bumped the minimum version to Dart 3.3.
